app/src/main/java/de/danoeh/antennapod/fragment/ItemFragment.java

```diff
@@ -319,9 +319,7 @@ private void updateButtons() {
         if (media == null) {
             actionButton1 = new MarkAsPlayedActionButton(item);
             actionButton2 = new VisitWebsiteActionButton(item);
-            noMediaLabel.setVisibility(View.VISIBLE);
         } else {
-            noMediaLabel.setVisibility(View.GONE);
```

Comment:
Looks like this unrelated code somehow got removed. Not sure what happened there. Could you please revert?